src/docquery/ext/pipeline_image_classification.py

```diff
@@ -0,0 +1,200 @@
+# This file is copied from transformers:
+#    https://github.com/huggingface/transformers/blob/bb6f6d53386bf2340eead6a8f9320ce61add3e96/src/transformers/pipelines/image_classification.py
+# And has been modified to support Donut
+import re
+from typing import List, Union
+
+from transformers.pipelines.base import PIPELINE_INIT_ARGS, Pipeline
+from transformers.utils import (
+    ExplicitEnum,
+    add_end_docstrings,
+    is_tf_available,
+    is_torch_available,
+    is_vision_available,
+    logging,
+    requires_backends,
+)
+
+
+if is_vision_available():
```

Comment:
Seeing as the following 3 helper functions are identical to the ones in the transformers library, I think we should just import theirs, unless there's an explicit reason for duplicating them.

Reply:
It's just for simplicity of maintaining this file (as a "snapshot" in time of theirs).